### PR TITLE
Bulk delete functionality

### PR DESCRIPTION
**Pull request by ivermac**

### CODE REVIEW

File: src/hatti/views/table.cljs [hunk 1]
```diff
@@ -187,6 +199,72 @@
...
      (.subscribe event handler-function))))
 
+(defn add-element
+  [vector value]
+  (-> vector
+      (conj (int value))
+      set
+      vec))
+
+(defn remove-element
```
Comment (pld) [inline src/hatti/views/table.cljs]: Rather generic helper could potentially go in Chimera

File: src/hatti/views/table.cljs [hunk 2]
```diff
@@ -187,6 +199,72 @@
...
+
+(defn check-select-unselect-all-records-element?
+  []
+  (let [delete-record-class-selector (str "." delete-record-class)
+        total-rows-count (get-elements-count-by-selector
+                          delete-record-class-selector)
+        ; TODO: test using (:data-to-be-deleted @shared/app-state)
+        selected-rows-count (get-elements-count-by-selector
+                             (str delete-record-class-selector ":checked"))]
+    (if (= selected-rows-count total-rows-count) true false)))
```
Comment (pld) [inline src/hatti/views/table.cljs]: Equality returns a Boolean

Comment (ivermac) [reply src/hatti/views/table.cljs]: 🤦‍♂️ i'll change that

File: src/hatti/views/table.cljs (hunk 1 above)
Comment (ivermac) [reply src/hatti/views/table.cljs]: Cool, I'll move both `add-element` and `remove-element` to Chimera

**Comment by ivermac** [comment]
I'll add the tests

**Pull request merged**